src/main/java/com/connexta/store/controllers/StoreController.java

```diff
@@ -121,7 +120,7 @@ public StoreController(@NotNull final StoreService storeService) {
 
       log.warn("Unable to retrieve {}", productId, e);
       return new ResponseEntity<>(HttpStatus.INTERNAL_SERVER_ERROR);
-    } catch (Throwable t) {
+    } catch (Exception exception) {
```

Comment:
SonarLint complained about catching Throwable:

> Code smell
> Major
> squid:S1181
> Throwable is the superclass of all errors and exceptions in Java. Error is the superclass of all errors, which are not meant to be caught by applications. Catching either Throwable or Error will also catch OutOfMemoryError and InternalError, from which an application should not attempt to recover.

Reply:
❗️We need to catch throwable here to make sure that the input stream gets closed. That should be suppressed.